software/controller/lib/core/nvparams.h

```diff
@@ -18,10 +18,15 @@ limitations under the License.
 #include <stddef.h>
 #include <stdint.h>
 
+#include <array>
+
 #include "eeprom.h"
 #include "network_protocol.pb.h"
 #include "units.h"
 
+// pinch_valve calibration size, defined here because the tables are stored in NVParams.
+static constexpr size_t pinch_valves_cal_size{11};
```

Comment:
Is this really the best place to define this? Maybe we need something like a `system_constants.h`?